contracts/GameJam.sol

```diff
@@ -60,19 +93,20 @@ contract GameJam {
   // finish function is a payable used to declare the winner via their address
   // the function should only be called able when the GameJam is in Registration
   function start() public payable
-    onlyAtStage(Stages.Registration)
+    onlyGameJamAdmin //Only an admin can start the GameJam
```

Comment:
How should payout be restricted? Only by an admin ?

Reply:
Yeah only the GameJam admin! ❤️